### ISSUE TITLE
Make sorting case-insensitive

### ISSUE THREAD

**Issue by irvnriir** [leftstick/vscode-favorites#81]
![image](https://github.com/user-attachments/assets/2b9779d9-96e9-49cf-84f6-0ece85a6a71f)

Almost every list uses case-insensitive sorting, the current behavior creates inconsistency.

**Comment by aRegularHumanBartender** [comment]
Yup. I second this. It's much easier to find if it's case-insensitive.
